doc/sphinx/source/conf.py

```diff
@@ -277,3 +279,11 @@
 
 # If true, do not generate a @detailmenu in the "Top" node's menu.
 #texinfo_no_detailmenu = False
+
+
+# Monkey patch to disable nonlocal image warning for Chapel logo
+def _warn_node(self, msg, node):
+    if not msg.startswith('nonlocal image URI found:'):
+        self._warnfunc(msg, '%s:%s' % get_source_line(node))
+
+sphinx.environment.BuildEnvironment.warn_node = _warn_node
```

Comment:
This is mostly @Kyle-B's  idea, but I don't think this should be calling _warnfunc since it's an internal function that could disappear without warning. 


Can you do something like:

```python
# remove "from docutils.utils import get_source_line"

original_warn_mode = sphinx.environment.BuildEnvironment.warn_node

def allow_nonlocal_image_warn_node(self, msg, node):
    if not msg.startswith('nonlocal image URI found:'):
        original_warn_mode(self, msg, node)

sphinx.environment.BuildEnvironment.warn_node = allow_nonlocal_image_warn_node
```


FYI it looks like an open PR for sphinx is adding a more principled way to ignore certain warnings (though clearly we won't be able to use it until it's included in a release.) Might be worth noting that there might be better way to do this in the future. 


https://github.com/sphinx-doc/sphinx/issues/1895

https://github.com/sphinx-doc/sphinx/pull/1969/files